chall02/agarzon-.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+
+morse = {
+	" ": " ",
+	"a": ".-",
+	"A": ".-",
+	"b": "-...",
+	"B": "-...",
+	"c": "-.-.",
+	"C": "-.-.",
+	"d": "-..",
+	"D": "-..",
+	"e": ".",
+	"E": ".",
+	"f": "..-.",
+	"F": "..-.",
+	"g": "--.",
+	"G": "--.",
+	"h": "....",
+	"H": "....",
+	"i": "..",
+	"I": "..",
+	"j": ".---",
+	"J": ".---",
+	"k": "-.-",
+	"K": "-.-",
+	"l": ".-..",
+	"L": ".-..",
+	"m": "--",
+	"M": "--",
+	"n": "-.",
+	"N": "-.",
+	"o": "---",
+	"O": "---",
+	"p": ".--.",
+	"P": ".--.",
+	"q": "--.-",
+	"Q": "--.-",
+	"r": ".-.",
+	"R": ".-.",
+	"s": "...",
+	"S": "...",
+	"t": "-",
+	"T": "-",
+	"u": "..-",
+	"U": "..-",
+	"v": "...-",
+	"V": "...-",
+	"w": ".--",
+	"W": ".--",
+	"x": "-..-",
+	"X": "-..-",
+	"y": "-.--",
+	"Y": "-.--",
+	"z": "--..",
+	"Z": "--.."
+}
+
+def is_alpha(s):

```

Comment:
Good idea of function, with it you didn't used `isascii` or `isalpha` functions and could check the characters required